cli/commands/apikey/help.go

```diff
@@ -0,0 +1,24 @@
+package apikey
+
+import (
+	"github.com/sensu/sensu-go/cli"
+	"github.com/spf13/cobra"
+)
+
+// HelpCommand defines new parent
+func HelpCommand(cli *cli.SensuCli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "apikey",
```

Comment:
I suggest we use `api-key` here and in other subcommands since we did the same for the other compound words (e.g. `cluster-role-binding`). WDYT?

Reply:
Ah, we don't use dashes in the api endpoint, but we do in the sensuctl command. Yes, I'll update that!